completions/configure

```diff
@@ -37,6 +37,6 @@ _configure()
         [[ ${COMPREPLY-} == *= ]] && compopt -o nospace
     fi
 } &&
-    complete -F _configure configure
+    complete -o bashdefault -o default -F _configure configure
```

Comment:
While I agree with:
> It seems reasonable in general to fall back on default completions if option completions didn't work.

...for this particular completion, the problem with the above implementation is that we don't have a way to say what worked or not. For example, we explicitly want to prevent all completions following --help, --version, --program-suffix and friends, but if we slap the defaults in the mix, our prevention will no longer work. I wish there was a way to signal success/not someway to prevent the defaults from kicking in (e.g. via some return value), but I'm not aware of a way.

Reply:
I would rather have possible superfluous completions after --help than have the completion of a directory not actually complete after an option that the completion doesn't recognize, but wants a directory anyway. The former is just a nuisance, the latter leads to copypaste typos that instead of failing the configure itself will fail the subsequent build.

Reply:
Mileages vary. Anyway I think it's possible to accomplish the original intent without sacrificing non-completions by not touching `complete` options at all, but doing this:
```diff
--- a/completions/configure
+++ b/completions/configure
@@ -25 +25,4 @@ _configure()
-    [[ $cur != -* ]] && return
+    if [[ $cur != -* ]]; then
+        _filedir
+        return
+    fi
```

Reply:
I verified that that works; plz patch that in. :)